app/client/components/wizardRouterAdapter.tsx

```diff
@@ -131,7 +120,7 @@ const ThisStageContent = (props: WizardStepProps) => (
 export interface WizardStepProps extends CommonProps {
   routeableStepProps: RouteableStepProps;
   children: any;
-  fullWidth?: true;
+  fullWidth: true;
```

Comment:
I think we can eliminate this prop now, since it appears to be present on all usages.